knip.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://unpkg.com/knip@5/schema.json",
+  "ignore": ["src/nodes/*.js"],
+  "ignoreFiles": [
+    "tests/format/RespectDefaultOptions/respect-default-options.js",
+    "tests/integration/node.test.js"
+  ],
```

Comment:
The difference between `ignore` and `ignoreFiles` is not super obvious:

- `ignore` means "don't report unused code inside this file".
- `ignoreFiles` means "don't report the whole file as unused"